src/js/internal/fs/cp.ts

```diff
@@ -17,7 +17,6 @@
 const { chmod, copyFile, lstat, mkdir, opendir, readlink, stat, symlink, unlink, utimes } = require("node:fs/promises");
 const { dirname, isAbsolute, join, parse, resolve, sep } = require("node:path");
 
-const SafePromiseAll = Promise.all;
 const PromisePrototypeThen = Promise.prototype.then;
```

Comment:
The hilarious thing about this is `.$then` exists and so it's less safe than the actual